src/initialize/initialize.ts

```diff
@@ -21,8 +22,10 @@ export function initialize(props: IInitializeProps): void{
     const {alternative_payment_methods} = getOrderInitialData();
     setOnAction(props.onAction);
 
-    if(alternative_payment_methods){
-        alternative_payment_methods.forEach(paymentMethod => {
+    if (props.fastlane) {
+        initFastlane();
+    } else if (alternative_payment_methods){
```

Comment:
Question, If we have other alternative payment methods along side fastlane don't we want to load them? 

If yes this peace is preventing that from happening.